AppCheckCore.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'AppCheckCore'
-  s.version          = '0.1.0-alpha.1'
+  s.version          = '0.1.0-alpha.4'
```

Comment:
Note: already pushed [`0.1.0-alpha.2`](https://github.com/firebase/SpecsDev/blob/57d1072c5adb86ebfeaf1a03f5bd212dc5805c96/AppCheckCore/0.1.0-alpha.2/AppCheckCore.podspec) and [`0.1.0-alpha.3`](https://github.com/firebase/SpecsDev/tree/main/AppCheckCore/0.1.0-alpha.3) to `SpecsDev` for other breaking API changes.